vm-linuxagent-upgrade/README.md

```diff
@@ -0,0 +1,25 @@
+# Create a Linux Virtual Machine with new version of Azure Linux Agent using Custom Script Linux Extension
```

Comment:
I'm really glad someone has written this template, but it needs to be clear that this is a snippet that shows how to update the extension on a Linux VM and not just something that is standalone. We really need to fix the extension so that it upgrades itself like it does on Windows, so in the meantime good to point out this is a workaround. 

There are a bunch of params that are unnecessary. I realize you copied them from our examples, which are unfortunately incorrect. You can hardcode most of the names for example (they are RG unique)